### ISSUE TITLE
countDocuments() helper

### ISSUE THREAD

**Issue by vkarpov15** [Automattic/mongoose#6643]
Now that count() is deprecated.

**Comment by ceo** [comment]
For those who are looking into it: 

http://mongoosejs.com/docs/api.html#countdocuments_countDocuments

**Comment by AbdelrahmanHafez** [comment]
There's probably something wrong with the docs now

```
Model.countDocuments()

Counts number of matching documents in a database collection. 
This method (count? countDocuments?) is deprecated, use countDocuments() instead.
```

**Comment by vkarpov15** [comment]
Good catch, we fixed that in edb91b2 but haven't released it yet. Will ship 5.2.2 later today.
